spiceaidocs/docs/clients/Datadog/index.md

```diff
@@ -11,34 +11,16 @@ Spice can be monitored with [Datadog](https://www.datadoghq.com/) using the [Spi
 
 ## Metrics Endpoint Configuration
 
-The metrics endpoint uses port `9000` by default. The metrics endpoint configuration is logged at startup.
+The metrics endpoint uses port `9090` by default. The metrics endpoint configuration is logged at startup.
 
 ```bash
-2024-07-15T21:48:00.158267Z  INFO spiced: Metrics listening on 127.0.0.1:9000
-```
-
-Pass the `--metrics` parameter to bind to a specific port. For example, to bind to port `9090`:
-
-```bash
- spiced --metrics 0.0.0.0:9090
-```
-
-or when using Docker:
-
-```Dockerfile
-FROM spiceai/spiceai:latest
-
-# Docker configuration ...
-
-# Configure the metrics endpoint on port 9090
-CMD ["--metrics", "0.0.0.0:9090"]
```

Comment:
This part is still required as we don't have metrics enabled by default when running Docker.